tensorflow/python/data/benchmarks/benchmark_base.py

```diff
@@ -113,8 +113,10 @@ def run_and_report_benchmark(self,
                                warmup=True,
                                apply_default_optimizations=False):
     # Measure the per-element wall time.
-    wall_time = self.run_benchmark(dataset, num_elements, iters, warmup,
-                                   apply_default_optimizations)
+    wall_time = self.run_benchmark(
```

Comment:
It would be great if you add a docstring for this method that documents the behavior (including the new behavior of returning the wall time).

Reply:
addressed in the commit.